### PR TITLE
Lesson 1.3

### PR DESCRIPTION
**Pull request by larasodjati**
Moved the list Item into the new "Todo List Item" component, handled "Add Todo Form" submit, and stored "New Todo" in React state

### CODE REVIEW

File: src/AddTodoForm.js
Comment (Atos20) [inline src/AddTodoForm.js]: I would just move this line `event.preventDefault` to line five to avoid any unintended behavior. This is my personal opinion because by putting int at the top of the function I don't have to think about the order of operations.

**Review by Atos20 — APPROVED** [review]
Great work!!!

I loved the commit history, this is exactly what is going to help you highlight your work !
The only thing I would ask you to do is to move the line `event.preventDefault()` to the top f your function before merging this PR. After that you won't need my approval, just go ahead and merge it. Thank you, great job ✅ !

File: src/AddTodoForm.js
Comment (larasodjati) [reply src/AddTodoForm.js]: Thank you so much for the feedback! 
I will push event.preventDefault to line 5 before merging it to the main branch.

**Pull request merged**
